src/UI/UIView.cpp

```diff
@@ -45,596 +54,905 @@ void UIView::draw_ui_callback()
+        if (ImGui::Button("Reset##Reset2")) {
+          m_clip_ypr = glm::vec3(0.0f);
+          m_clip_pos = glm::vec3(0.0f);
+        }
 
-      ImGui::Checkbox("use transferfunction", &m_use_transferfunction);
-      if (m_use_transferfunction)
-      {
+      }
+      ImGui::EndTabItem();
+    }
 
+    if (ImGui::BeginTabItem("Animation"))
+    {
 
-        bool is_multi_channel = m_controller_app.data_is_multi_channel();
-        if (is_multi_channel)
-        {
-          for (int i = 0; i < 3; i++) {
-            if (m_animated)
-            {
-              /*
-                frame by frame animation
+      ImGui::Text("Time Frames");
 
-                unsigned int active_volume = floor(m_frame);
-                unsigned int active_volume2 = ceil(m_frame);
-                double alpha = m_frame - active_volume;
-                if (active_volume < m_volumes[m_selectedVolume].size() && active_volume2 < m_volumes[m_selectedVolume].size())
-                {
-                  tfn_widget_multi[m_selectedVolume].setBlendedHistogram(
-                    m_volumes[m_selectedVolume][active_volume]->getTransferfunction(i),
-                    m_volumes[m_selectedVolume][active_volume2]->getTransferfunction(i), alpha, i);
-                }*/
+      if (ImGui::Button("Add")) {
+        m_camera_name_window_open = true;
+        m_camera_button_action = BUTTON_ACTION::ADD;
 
-            }
-            else {
-              /*		tfn_widget_multi[m_selectedVolume].setHistogram(m_volumes[m_selectedVolume][0]->getTransferfunction(i), i);*/
-            }
+      }
+      ImGui::SameLine();
+      if (ImGui::Button("Remove")) {
+        if (m_controller_app.get_simulation().get_simulation_states().size() > 0)
+        {
+          m_controller_app.get_simulation().remove_simulation_state(m_simulation_state_selection);
+          if (m_simulation_state_selection - 1 < 0)
+          {
+            m_simulation_state_selection = 0;
           }
-          m_controller_app.set_multi_transfer(true);
-          //tfn_widget_multi[m_selectedTrnFnc].draw_histogram();
-          tfn_widget_multi[m_table_selection].draw_ui();
+          else
+          {
+            m_simulation_state_selection--;
+            SimulationState next_sim_state = m_controller_app.get_simulation().get_simulation_state_at(m_simulation_state_selection);
+            m_controller_app.get_trackball_camera().set_current_poi(next_sim_state.poi);
+          }
+
+
         }
-        else
+
+      }
+      ImGui::SameLine();
+      if (ImGui::Button("Reset Camera")) {
+        m_controller_app.get_trackball_camera().reset_camera();
+      }
+      ImGui::BeginTable("##Camera Point of interest (POI) Editor", 1, ImGuiTableFlags_Borders);
+      ImGui::TableSetupColumn("Name", ImGuiTableColumnFlags_WidthFixed, 120.0f);
+
+
+      for (int row = 0; row < m_controller_app.get_simulation().get_simulation_states().size(); ++row)
+      {
```

Comment:
can these rows be easily reordered?

Reply:
What do you mean by reordered? If it is moving rows up/down, then no.